views/aggregate-flow-metadata.json

```diff
@@ -2281,11 +2321,15 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-hex",
             "description": {
                 "summary": "Flow that waits for the triggered project run to complete.",
                 "returns": "The status of the project run and the metadata associated with the run."
-            }
+            },
+            "documentation_url": "https://prefecthq.github.io/prefect-hex/project/#prefect_hex.project",
```

Comment:
Should the documentation url point to the actual flow object?
https://prefecthq.github.io/prefect-hex/project/#prefect_hex.project.wait_for_project_run_completion

Reply:
is there an actual link to point at for the flow object in the docs?

Reply:
https://prefecthq.github.io/prefect-hex/project/#prefect_hex.project.wait_for_project_run_completion

Reply:
nice - will do